cloudplatform/connectivity-destination-service/src/test/java/com/sap/cloud/sdk/cloudplatform/connectivity/DestinationServiceFactoryTest.java

```diff
@@ -100,9 +101,14 @@ void testHttpDestination()
         assertThat(result.get(DestinationProperty.NAME)).contains("httpDestination");
         assertThat(result.get(DestinationProperty.URI)).contains("https://example.com");
         assertThat(result.get(DestinationProperty.AUTH_TOKENS).isDefined()).isTrue();
-        assertThat(result.get(DestinationProperty.AUTH_TOKENS).get()).asList().containsExactly(token);
+        assertThat(result.get(DestinationProperty.AUTH_TOKENS).get())
+            .asInstanceOf(InstanceOfAssertFactories.LIST)
+            .isNotEmpty()
+            .containsExactlyInAnyOrder(token);
```

Comment:
Why did you choose containsExactlyInAnyOrder for this assertion but containsExactly for the one below?
```suggestion
            .containsExactly(token);
```